docs/userguide.rst

```diff
@@ -162,8 +162,12 @@ file that comes with the package:
         environment:
           - ZMQ_PROXY_CONNECT: "tcp://zeromq-msg-proxy.default.svc.cluster.local:8666"
 
-You can use the supplied ``reana-cluster.yaml``, or create use ``-f``
-command-line option and specify your own file.
+You can use the supplied ``reana-cluster.yaml``, or create your own and use the ``-f``
+command-line option and specify your own file, in the following way:
+
+.. code-block:: console
+
+  $ reana-cluster -f reana-cluster-custom.yaml init
```

Comment:
People will learn about the `init` command in the next section only, as it were, so what about keeping the narrative progressive, and in this part you can mention that people may want to copy the default file and customise it -- for example you can show the creation of `reana-cluster-custom.yml` -- and mention the `-f` part only at the end of the next section, something like "If you have created your custom configuration, you can use it as follows..."?

Reply:
Yes sounds good.